### ISSUE TITLE
Merge stuff

### ISSUE THREAD

**Issue by lakruzz** [lakruzz/sc1#6]
Let's play a game: You are now the _integration manager_ of a software project.

It's actually a title used (less frequently). It's the person who does all the merges - sometimes also referred to as _merge manager_ or simply _"The Merge Guy"_. Does that sound like a fun job? Luckily it's not as common as it used to be thanks to tools like git and GitHub and to the spread of more agile processes - most developers today are expected to merge their own stuff and and push it.

But the whole _Pull request_ culture - which is very popular, and advocated by GitHub is kind of a relic from the past. A distributed approach to an old school integration manager.

We'll touch on this later when we talk about pull requests.

**Your job is now to make all the branches of the poem come together as one on the `master` branch**

To Recap the situation:

- @denvercoder changed _tiger_ to _lion_ on the `lion` branch
- @wwhite: changed _mother_ to _father_ on the `father` branch
- @Julesw: Changed _tiger_ to _turtle_ on the `turtle` branch
- YOU: Change an exclamation mark a punctuation mark directly on the `master`.

**Comment by lakruzz** [comment]
```mermaid
%%{init: { 'logLevel': 'debug', 'theme': 'base', 'gitGraph': {'showBranches': true, 'showCommitLabel':true,'mainBranchName': 'MetroLine1'}} }%%
      gitGraph
        commit id:"NewYork"
        commit id:"Dallas"
        branch MetroLine2
        commit id:"LosAngeles"
        commit id:"Chicago"
        commit id:"Houston"
        branch MetroLine3
        commit id:"Phoenix"
        commit type: HIGHLIGHT id:"Denver"
        commit id:"Boston"
        checkout MetroLine1
        commit id:"Atlanta"
        merge MetroLine3
        commit id:"Miami"
        commit id:"Washington"
        merge MetroLine2 tag:"MY JUNCTION"
        commit id:"Boston"
        commit id:"Detroit"
        commit type:REVERSE id:"SanFrancisco"
```

**Comment by lakruzz** [comment]
<img width="268" alt="image" src="https://user-images.githubusercontent.com/155492/218119030-12dc6192-e505-4393-9b3c-68eab2af634c.png">
